src/Stryker.CLI/Stryker.CLI/Stryker.CLI.csproj

```diff
@@ -1,32 +1,36 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.1</TargetFramework>
-    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
-    <Authors>Richard Werkman</Authors>
-    <Product>Mutation Testing</Product>
-    <AssemblyName>dotnet-stryker</AssemblyName>
     <RootNamespace>Stryker.CLI</RootNamespace>
-    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <AssemblyName>Stryker.CLI</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>dotnet-stryker</ToolCommandName>
+  </PropertyGroup>
+  
+  <PropertyGroup>
     <VersionPrefix>0.9.0</VersionPrefix>
-    <PackageId>StrykerMutator.DotNetCoreCli</PackageId>
-    <Description>The Stryker.NET Command Line Interface Runner for .NET Core. Adds the dotnet stryker command to your test project.</Description>
+    <PackageId>dotnet-stryker</PackageId>
+    <Authors>Richard Werkman, Rouke Broersma</Authors>
+    <Product>Mutation Testing</Product>  
+    <Description>The Stryker.NET dotnet CLI SDK runner. Install this cli runner with dotnet tool install -g dotnet-stryker.</Description>
```

Comment:
Why "SDK runner"? I don't think we are a SDK... Suggestion:

The Stryker.NET CLI tool. Install this tool to add the `dotnet stryker` command to your system.

Reply:
👍